tapgarden/caretaker.go

```diff
@@ -30,8 +31,10 @@ import (
 var (
 	// GenesisDummyScript is a dummy script that we'll use to fund the
 	// initial PSBT packet that'll create initial set of assets. It's the
-	// same size as a encoded P2TR output.
-	GenesisDummyScript [34]byte
+	// same size as a encoded P2TR output and has a valid P2TR prefix.
+	GenesisDummyScript = append(
+		[]byte{0x51, 0x20}, bytes.Repeat([]byte{0x00}, 32)...,
```

Comment:
Ah issue was it wasn't being recognized as proper P2TR before? Then possilby fell through to some default I guess. 

I think we can make this a bit more readable as: 
```go
GenesisDummyScript []byte{txscript.OP_1, ...}
```

Or we could even make a custom script builder, but `OP_1` makes it a bit easier to parse imo.

Reply:
> Then possilby fell through to some default I guess.

If this is the case, can we add to the commit message?

Reply:
An example of the builder route: https://github.com/lightningnetwork/lnd/blob/master/input/taproot.go#L144-L149C25

Reply:
Yeah I think it was just misclassified but Oli found the alt script type, I don't recall it exactly. Will add notes to commit msg tho.